frontend/src/layers/AdministrativeLayers.js

```diff
@@ -54,4 +59,6 @@ const AdministrativeLayers = ({ map }) => {
   return null
 }
 
+// TODO This is strange and I don't think it works since hooks are inner-component dependant.
+// Either memoize it within the parent components or maybe memoize only its props values.
 export default React.memo(AdministrativeLayers)
```

Comment:
I think I made some tests, but I can retry
https://dmitripavlutin.com/use-react-memo-wisely/

Reply:
That's an interesting utility indeed that I didn't know. I will test it aside to fully check that VS classic mix of memo / callback hooks. Re-rendering is supposed not to be an issue in diff dom if this render the same node 'ids' since the browser only re-render the differences. But I admit that I don't know what React does withe de same prop values, I know it's a shallow equality check so there is that... It's a sort of a `shouldComponentUpdate()` remake from what I understand.

Reply:
I will remove my comment because it's indeed wrong, and fully understand that ^^.